hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerManagerV2.java

```diff
@@ -113,7 +90,7 @@ void deleteContainer(ContainerID containerID)
    */
   void updateContainerState(ContainerID containerID,
                             LifeCycleEvent event)
-      throws ContainerNotFoundException;
+      throws IOException, InvalidStateTransitionException;
```

Comment:
NIT: javadoc update @throws

Reply:
I'm planning to add detailed javadoc in follow up jiras. Will fix the @throws here.